### DISCUSSION TITLE
유니티 publish 오류 문의드립니다!

### DISCUSSION THREAD

**Discussion by PINK-J** [naverz/zepeto-studio-kor#1040] [5. Q&A]
<font class="papago-parent"><font class="papago-source" style="display:none;">안녕하세요..  월드 제작 후에 publish을 하니 오류가 생기면서 되지않습니다..</font>😣
</font>
</font><font class="papago-parent"><font class="papago-source" style="display:none;">처음에는 오브젝트 문제인가 싶어서 새로운 씬을 만들고, 오브젝트를 옮겨서도 해봤는데 계속 오류가 생기길래..</font>
</font><font class="papago-parent"><font class="papago-source" style="display:none;">
</font>새로 씬을 만들어서 그 상태로 publish을 해봤는데도 오류가 발생합니다..</font><font class="papago-parent"><font class="papago-source" style="display:none;"> 캡처화면 첨부 합니다😭
</font><font class="papago-parent"><font class="papago-source" style="display:none;">
</font>![filename1](https://user-images.githubusercontent.com/109596692/179746573-a51a092d-ba71-4c80-88e5-87828ecb152b.png)</font><font class="papago-parent"><font class="papago-source" style="display:none;">
</font>![http2](https://user-images.githubusercontent.com/109596692/179746586-08ddf711-d28a-47fb-8ec8-555e58db7464.png)</font><font class="papago-parent"><font class="papago-source" style="display:none;">
</font>
</font>

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 멀티플레이 코드 및 스키마 오류로 보여집니다.
멀티플레이를 걷어낸 후 빌드를 해서 오류가 없는지 우선 살펴보신 후
아래 공식 가이드에 따라 멀티플레이를 하나씩 구현해 나가시면 도움이 되실 것 같습니다.
https://github.com/naverz/zepeto-studio-kor/discussions/459
